src/components/FilterPanel/FilterPanel.stories.js

```diff
@@ -162,7 +162,8 @@ storiesOf(patterns('FilterPanel'), module)
       </FilterPanelAccordion>
       <FilterPanelAccordion title="Filter accordion 2" count={100}>
```

Comment:
Do the `FilterPanelAccordion` or `FilterPanelGroup` props also need updates to the prop types?

Reply:
It looks like `FilterPanelAccordion` just passes the `title` prop value to `FilterPanelGroup`:
https://github.com/carbon-design-system/ibm-security/blob/9b621a92fc10b0651be3d4f5dee1db8bbc2b2f3c/src/components/FilterPanel/FilterPanelAccordion/FilterPanelAccordion.js#L24-L30

And `FilterPanelGroup` just sends it to `FilterPanelLabel`, which is addressed in this PR:
https://github.com/carbon-design-system/ibm-security/blob/9b621a92fc10b0651be3d4f5dee1db8bbc2b2f3c/src/components/FilterPanel/FilterPanelGroup/FilterPanelGroup.js#L27-L29

If you mean the propType documentation, sure that can be updated to be more specific. 👍

Reply:
That and also the `propTypes` of the components probably need updates:

https://github.com/carbon-design-system/ibm-security/blob/cf0819c334010dd430319482d56ec9941d729d39/src/components/FilterPanel/FilterPanelAccordion/FilterPanelAccordion.js#L35-L49

Reply:
Yes, certainly -- if we are going with the changes you proposed in the other comment, then I will of course be updating the propType of the `title` prop to keep in it line with my original proposed fix: that is, not passing a `node` to the HTML `title` attribute 👍 

And I will also be adding propType information with a new `heading` prop 👍